internal/apis/bootstrap/kubeadm/v1alpha3/conversion_test.go

```diff
@@ -91,3 +92,12 @@ func kubeadmBootstrapTokenStringFuzzerV1Beta1(in *bootstrapv1.BootstrapTokenStri
 	in.ID = "abcdef"
 	in.Secret = "abcdef0123456789"
 }
+
+func bootstrapv1JoinConfigurationFuzzer(obj *bootstrapv1.JoinConfiguration, c fuzz.Continue) {
```

Comment:
Conversion must work without this. We are converting here between v1alpha3 <=> v1beta1 

(also potentially the reason why the potential panic in the conversion code was not found)

(same for v1alpha4)